linux_proc_extras/datadog_checks/linux_proc_extras/linux_proc_extras.py

```diff
@@ -24,10 +24,12 @@
 
 
 class MoreUnixCheck(AgentCheck):
-    def check(self, instance):
-        self.tags = instance.get('tags', [])
+    def __init__(self, *args, **kwargs):
+        super(MoreUnixCheck, self).__init__(*args, **kwargs)
+        self.tags = self.instances[0].get('tags', [])
```

Comment:
We also have access to `self.instance` that does the same, a bit more obvious
```suggestion
        self.tags = self.instance.get('tags', [])
```